drivers/python/tox.ini

```diff
@@ -0,0 +1,26 @@
+[tox]
+skipsdist = True
```

Comment:
I don't think that `skipdist` should be set. `sdist` is very cheap for us, and with some recent changes that would make it even better to test with `sdist`-trimmed drivers.